packages/components/src/MultiSelect/types.ts

```diff
@@ -1,5 +1,12 @@
+import { FieldMessageStatus } from "~components/FieldMessage"
+
 export type MultiSelectOption = {
   label: string
   // Used for "value" attribute in option - must be unique
   value: string | number
 }
+
+export type ValidationMessage = {
+  status?: Extract<FieldMessageStatus, "error" | "caution">
+  message: string | React.ReactElement
+}
```

Comment:
Seems like something we need to make more widely accessible to the other fields

Reply:
You mean like a shared types file in the root of the src similar to the style utils?

Reply:
Could it belong to a component?

Reply:
We could export the type from FieldMessage since these props are consumed in that component - assuming that we only want validation messages to have `error` and `caution` states.

Reply:
We can always extend later, but its your call whether it makes sense to do it now 😄

Reply:
My main concern was the existing one in the Filter Date Picker, since it has all states accounted for, but looking at the story its only ever used for a error state. As you say we can always extend on it and it doesn't hurt to be more specific in what we're expecting

Reply:
Hmmm gave it a shot and realized I'd have to make a fair few changes to the imports in the Filter DP. I think I'll revert that back and create a ticket following this one to move that type to the FieldMessage component once this is merged. I'd rather untangle this release and get a specific optimization out as its own thing :)